controllers/profiles/reconciler_dev.go

```diff
@@ -51,6 +56,8 @@ const (
 	recoverDeploymentErrorRetries = 3
 	// recoverDeploymentErrorInterval interval between recovering from failures
 	recoverDeploymentErrorInterval = 5 * time.Minute
+	nightlySuffix                  = "nighlty"
+	snapshotSuffix                 = "snapshot"
```

Comment:
isn't better to use the snapshot version instead suffix? snapshot is 2.0.0-snapshot.

Reply:
is used to understand if a version is a snapshot